Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,611 @@
+via a factory mechanism based on the file extension
+([`itk::ImageIOFactory`](https://itk.org/Doxygen/html/classitk_1_1ImageIOFactory.html)). Some proprietary files format are only imported.
+More information on [ITK File Formats](https://itk.org/Wiki/ITK_File_Formats) is
+available, including data types supported.
+
+A very nice tool for viewing all of these images is [ITKApps/ImageViewer](https://github.com/InsightSoftwareConsortium/ITKApps/tree/master/ImageViewer).
+
+| File Format | Read/Write | Import |
+| ----------- | ---------- | ------ |
+| Analyze 7.5 | AnalyzeImageIO | |
+| BMP † | BMPImageIO | |
+| DICOM | GDCMImageIO | Using DicomImageIO is backward compatible, but user should prefer GDCMImageIO |
+| GE 4x | | GE4ImageIO |
+| GE 5x | | GE5ImageIO |
+| GE Advantage Windows | | GEAdwImageIO |
+| Gipl (.gipl) | GiplImageIO | |
+| IPLCommon | | IPLCommonImageIO |
+| JPEG † | JPEGImageIO | |
+| NRRD (.nhdr/.nrrd) | NrrdImageIO | |
+| Nifti (.nii) | NiftiImageIO | |
+| MetaImage (.mha/.mhd) | MetaImageIO | |
+| PNG † | PNGImageIO | |
+| Raw ‡ | RawImageIO | |
+| Siemens Vision | | SiemensVisionImageIO |
+| Stimulate (spr/sdt) | StimulateImageIO | |
+| TIFF | TIFFImageIO | |
+| VTK Structured Points | VTKImageIO | |
+
+† BMP, PNG and JPEG are not very good formats for 3D medical images.
+They only support 2D images and a limited set of pixel types such as
+unsigned char, and unsigned short. The great advantage of BMP, PNG and
+JPEG is that you can view them with almost any image viewer. It is
+possible to read/write 3D datasets as collections of 2D slices each one
+in an independent BMP, PNG or JPEG file by using the
+`ImageSerierReader` and [`itk::ImageSeriesWriter`](https://itk.org/Doxygen/html/classitk_1_1ImageSeriesWriter.html).
+
+‡ It is higly recommended not using this format. RAW is not a format, it
+is insufficient, inconsistent and simply dangerous. A RAW file without a
+header is useless, and once you add a header, it is not RAW anymore.
+
+### What if my file format is not supported by ITK?
+
+If ITK doesn't have a specific file format reader at this point, you
+may try converting this specific file format image to a format like PNG,
+for which ITK has readers.
+
+A good set of tools for performing such conversions is available in
+ImageMagick, which can be downloaded for free from https://imagemagick.org/index.php.
+
+You can also take advantage of the VTK readers/importers. All you need
+is then use the VTKImageToImage adaptor in order to convert the
+[`vtkImageData`](https://vtk.org/doc/nightly/html/classvtkImageData.html) into an [`itk::Image`](https://itk.org/Doxygen/html/classitk_1_1Image.html).
+
+Supported VTK file formats can be found in https://docs.vtk.org/en/latest/supported_data_formats.html.
+
+### How do I read a volume from a DICOM series?
+
+The following are alternative options for reading DICOM series. The
+first one in the list is the recommended option. The others are provided
+in case of desperation.
+
```

Comment:
I think option 0 here should be @itk-wasm dicom, which has the broadest support and is the easiest to use. [python docs](https://itk-wasm-dicom-python-docs.on.fleek.co/), [js docs](https://itk-wasm-dicom-docs.on.fleek.co/#/).

Reply:
Done in 894e5e6.